demos/smart_classroom_demo/action_event_metrics.py

```diff
@@ -174,13 +170,13 @@ def match_detections(predicted_data, gt_data, min_iou):
 
         matches = []
         visited_gt = np.zeros(len(gt_bboxes), dtype=np.bool)
-        for i in xrange(len(sorted_predicted_bboxes)):
+        for i in range(len(sorted_predicted_bboxes)):
```

Comment:
According usage of `range` function in the code it looks like we can eliminate the `range` function at all: replace `for i in range(len())` construction onto `for i, _ in enumerate()` and so on.

Reply:
I am a huge fan of __enumerate()__ but it’s use is too counter intuitive if we are not going to use each element.  My preference would be to stick with your first solution instead.

Reply:
Agree.